helpers/userEvents.js

```diff
@@ -0,0 +1,49 @@
+var MobileDetect = require('mobile-detect');
+var UserAnalytics = require('user-analytics');
+
+function UserEvents() {}
+
+UserEvents.trackSignup = function(req, user) {
+    const endpoint = process.env.USER_ANALYTICS_ENDPOINT;
+    const token = process.env.USER_ANALYTICS_TOKEN;
+    const streamName = process.env.USER_ANALYTICS_STREAM_NAME;
+    const env = process.env.NODE_ENV;
+    console.log('options', endpoint, token);
+    const userAnalytics = new UserAnalytics({
+        token: token,
+        endpoint: endpoint,
+    });
```

Comment:
this setup work can be done once during instantiation of this module, rather than each time it's called